string data_ip
---
bool result
